.teamcity/src/main/kotlin/model/CIBuildModel.kt

```diff
@@ -328,9 +328,9 @@ enum class PerformanceTestType(
     historical(
         displayName = "Historical Performance Test",
         timeout = 2280,
-        defaultBaselines = "3.5.1,4.10.3,5.6.4,6.9.1,last",
+        defaultBaselines = "last",
```

Comment:
I think it would be good to make this configurable somehow, so you could run the historical performance tests against a given baseline on CI. For example, I'd like to see how the current Gradle version compares to Gradle 6.9.1.

Is this already possible? Or do we need to introduce a parameter for `defaultBaselines` on the performance test jobs?

Reply:
I think we can make it configurable with `last` as default value.